packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx

```diff
@@ -52,7 +65,7 @@ export const TableField = <TData extends object>({ label, data, onChange, column
   };
 
   const showAddButton = () => {
-    if (tableData.filter(obj => deepEqual(obj, emptyDataObject)).length === 0) {
+    if (tableData.filter(obj => deepEqual(obj, emptyDataObject)).length <= 1) {
```

Comment:
<==

Reply:
for some reason this does not work:
![grafik](https://github.com/user-attachments/assets/aad2866a-42b4-4c26-970d-040df3b35a5e)